README.md

```diff
@@ -1,6 +1,74 @@
 # pxt-rovercode
 
+## Development Usage
 
+```bash
+$ sudo npm install -g pxt
+$ pxt target microbit
+$ pxt install
+$ pxt build # output at built/binary.hex
+```
+
+## Protocol
+
+The following protocol is used over the Nordic virtual BLE serial channel.
+
+The common format is `{identifier}:{comma-separated values}`.
+Messages from the webapp to the micro:bit end with a `\n`; messages from the micro:bit to the webapp do not.
+
+### Set motor power
+
+#### Direction
+webapp -> micro:bit
+
+#### Identifier
+`set-left-motor` and `set-right-motor`
+
+#### Data
+| Index    | Description                 | Type   | Unit              |
+|----------|-----------------------------|--------|-------------------|
+| 0        | Motor power                 | Number | 0-100             |
+
+#### Example
+```
+set-left-motor:0\n
+set-right-motor:100\n
+```
+
+### Sensor status
+
+#### Direction
+micro:bit -> webapp
+
+#### Identifier
+
+`sensor-status`
+
+#### Data
+
+| Index    | Description                 | Type   | Unit              |
+|----------|-----------------------------|--------|-------------------|
+| 0        | Left light sensor value     | Number | 0-255             |
```

Comment:
Might need clarification here as well

Reply:
Just added a blanket clarification in the top section.